packages/block-library/src/post-comments-form/style.scss

```diff
@@ -1,3 +1,18 @@
+// Allow these default styles to be overridden by global styles.
+:where(.wp-block-post-comments-form) {
+	textarea,
+	input:not([type="submit"]) {
+		border: 1px solid $gray-600;
+		font-size: 1em;
+		font-family: inherit;
+	}
+
+	textarea,
+	input:not([type="submit"]):not([type="checkbox"]) {
```

Comment:
These chained `:not()` will make the specificity of this rule be 0-2-1. If the goal is to level all the rules at 0-1-0, it might be better to wrap these nested selectors in `:where()` instead of the outer selector.